scripts/test.sh

```diff
@@ -79,6 +108,19 @@ test() {
         error=true
     fi
 
+    #check if debug is working
+    cat $DEVFILES_DIR"$devfileName/devfile.yaml" | grep debug
```

Comment:
It might be a little bit safer to check for `kind: debug` instead just `debug`

Reply:
Understood